eox_nelp/user_profile/api/v1/tests/test_views.py

```diff
@@ -141,3 +141,37 @@ def test_account_update_extra_fields(self, cdd_task_mock):
         self.assertEqual(self.user.first_name, payload["first_name"])
         self.assertEqual(self.user.last_name, payload["last_name"])
         cdd_task_mock.delay.assert_called_with(user_id=self.user.id)
+
+    @override_settings(
+        ENABLE_OTP_VALIDATION=False,
+        REQUIRED_USER_EXTRA_INFO_FIELDS=["arabic_first_name", "arabic_last_name"],
+        PEARSON_RTI_ACTIVATE_GRADED_GATE=True,
```

Comment:
IMO we could remove this since the test purpose is to check if the extrainfo records was updated instead of checking if the CCD was called

Reply:
The profile API does the same, but I think that the cdd_task could also be verified.

Reply:
https://github.com/eduNEXT/eox-nelp/blob/197409afada7c98402b876bc66e40154c4cc6197/eox_nelp/user_profile/api/v1/tests/test_views.py#L148-L161